library/testing/TestSDKExternalWindowQT.cxx

```diff
@@ -67,9 +67,11 @@ int TestSDKExternalWindowQT(int argc, char* argv[])
     std::string(argv[1]) + "/data/cow.vtp", std::string(argv[1]) + "/baselines/", argv[2]);
   w.setTitle("F3D QT External Window");
   w.resize(300, 300);
-  w.startTimer(1000);
+  int timerId = w.startTimer(1000);
```

Comment:
why is the leak appears in this PR but not on master?

Reply:
this PR fix sanitizer. It was incorrectly set since some PR ago.

Reply:
What is the right way to silence it ?